src/adaXT/random_forest/random_forest.py

```diff
@@ -36,7 +32,7 @@ def tree_based_weights(
         hash1=hash1,
         size_X0=size_X0,
         size_X1=size_X1,
-        scale=scale,
+        scaling=scaling,
```

Comment:
Should the scaling with a string input be outwards facing? Would you not always want row wise scaling when scaling tree_based_weights? Do we maybe want to make it a bool and then convert internally?

Reply:
Its the same function that is also used for the similarity which is again a different scaling so I think it needs to be outward facing here or not?